### DISCUSSION TITLE
How can I modify the DOM element of a sprite?

### DISCUSSION THREAD

**Discussion by NotFenixio** [TurboWarp/extensions#1292] [Q&A] [answered]
So, I'm trying to make a blur extension for sprites, and potentially other types of transformations supported in CSS. But, I need to somehow get access to the DOM element of the sprite itself. How can I do this?

**Comment by David-Orangemoon** [comment, marked as answer]
The sprite is not an element on the DOM. it is a set of triangles drawn by webgl sorry.
